docs/configuration.md

```diff
@@ -211,6 +211,30 @@ The resulting artifact is a **Maven** package that can be deployed to
 [Maven Centra]: https://search.maven.org
 [nexus-staging-maven-plugin]: https://mvnrepository.com/artifact/org.sonatype.plugins/nexus-staging-maven-plugin
 
+#### Configuring `Kotlin/JVM`
+
+Kotlin/JVM target configuration is [similar to Java](#configuring-java).
+* `maven` – `groupId`, `artifactId` and optional `versionSuffix`, as in Java.
+* `package` – the containing Kotlin qualified package name.
+
+Example:
+```js
+{
+  "jsii": {
+    "kotlin-jvm": {
```

Comment:
Would you consider a multi-platform binding? Kotlin JS might give us more ability to interact with Typescript/JS directly.

Reply:
Our plan is to have a Kotlin-JVM and a Kotlin-JS target (separate). Nothing carved in stone though... But we definitely intend on supporting both flavors in one way or another.